src/tests/tuits-service.test.js

```diff
@@ -205,4 +207,40 @@ describe('can retrieve all tuits with REST API', () => {
             expect(tuit.postedBy._id).toEqual(author._id)
         })
     })
-});
\ No newline at end of file
+});
+
+describe('can search tuit by tuit', () => {
```

Comment:
This one fails, I can try if I can fix this.
Also, could we test updating a tuit?

Reply:
Sure, but I didn't update that from the source code which is why I didn't write any additional tests for that.

Reply:
I didn't write tests for updateTuit in our starting code, because it is not required in previous assignments.